src/parser.ml

```diff
@@ -358,6 +360,7 @@ let parser tactic =
   | i:{_:_focus_ nat_lit}       -> Pos.in_pos _loc (P_tac_focus(i))
   | _print_                     -> Pos.in_pos _loc P_tac_print
   | _proofterm_                 -> Pos.in_pos _loc P_tac_proofterm
+  | _why3_ s:ident             -> Pos.in_pos _loc (P_tac_why3(s))
```

Comment:
The ident could be optional and a default prover used in this case. This could be set using the builtins mechanism.

Reply:
Is it better to use only a set command instead of a builtins like this example ?
```
// some lambdapi code ...
set prover altergo
// some lambdapi code ...
```

Reply:
Yes, it's fine for me.